docs/pages/plugins-manifest.md

```diff
@@ -10,11 +10,11 @@ excerpt: The manifest is a JSON file that provides information about a plugin su
 
 The manifest is a JSON file that provides information about a plugin such as author, description, icon and where to get the latest update from. It also tells Sketch how to run the plugin and how it integrates into the _Plugins_ menu.
 
-## Example manifest
+> If you're interested in validating a `manifest.json` file, or understanding the file structure in detail, we maintain a JSON Schema [here](https://github.com/BohemianCoding/SketchAPI/blob/615874d7a3d12a00b56fcbabdf1a568ef7bb0ad0/docs/sketch-plugin-manifest-schema.json)
 
-The example plugin below called _Select Shapes_ defines three commands _All_, _Circles_ and _Rectangles_ which can be accessed from _Plugins_ › _Select Shape_.
+## Example manifest
 
-It also includes an `appcast` URL for Sketch to automatically check if a new version of the plugin is available.
+The example `manifest.json` file below is for a plugin called _Select Shapes_, and defines three commands - _All_, _Circles_ and _Rectangles_. These commands would be accessible from the _Plugins_ › _Select Shape_ menu in Sketch.
 
```

Comment:
Two minor formatting issues:

- Instead of "…three commands - _All_…": "…three commands – _All_…" (replace hyphen) 
- ~_Select Shape_~ _Select Shapes_ menu in Sketch

Reply:
Oh man, I almost used an emdash too and thought surely Chrisitian won't notice that 😬